### DISCUSSION TITLE
Add the ability to change the pressure

### DISCUSSION THREAD

**Discussion by liqqquid** [niels747/2D-Weather-Sandbox#87] [Ideas]
![Screenshot 2023-06-18 2 42 55 PM](https://github.com/niels747/2D-Weather-Sandbox/assets/136999536/0719c4be-0b78-4bf7-9bf3-e9b5847dae0b)
I was just playing around when I noticed a sonic boom created by the land descending. 
So it got me thinking: could you add a setting where you can add/decrease pressure? If you could, it would be awesome. Because as it turns out, I am researching how shock waves/blast waves effect clouds and precipitation.

**Comment by coolkid0281** [comment]
Yeah! That would be cool, I know you can kind of change the pressure by turning the brush to 0.05 and then turn the temp brush on and then press Ctrl to make cold, and you can change the pressure that way
